packages/email/email.js

```diff
@@ -43,27 +48,73 @@ var makeTransport = function (mailUrlString) {
     mailUrl.query.pool = 'true';
   }
 
-  var transport = nodemailer.createTransport(
-    urlModule.format(mailUrl));
+  const transport = nodemailer.createTransport(url.format(mailUrl));
+
+  transport._syncSendMail = Meteor.wrapAsync(transport.sendMail, transport);
+  return transport;
+};
+
+// More info: https://nodemailer.com/smtp/well-known/
+const knownHostsTransport = function(settings = undefined, url = undefined) {
+  let service, user, password;
+  if (url && !settings) {
+    let host = url?.split(':')[0];
```

Comment:
why do we need ?

Reply:
This is to get known host from url like from this url that @xet7 wants to use: `Outlook365://firstname.lastname:password@hotmail.com`

Reply:
Ok, but why do we need `?` as you already checked if the url is "true"

Reply:
Remnant from previous version of the code. Fixing...